lxd/devices.go

```diff
@@ -1593,3 +1595,492 @@ func deviceLoadInfiniband() (map[string]IBF, error) {
 	// check whether the device is an infiniband device
 	return UseableDevices, nil
 }
+
+func deviceInotifyInit(s *state.State) (int, error) {
```

Comment:
The entire inotify stuff really should get its own file.

Reply:
I need a version of `containerLoadByName()` can call from a subpackage. Even if I give the inotify functions a callback - which is ugly in itself - I run into the issue that the return value of `containerLoadByName()` is `container` which is an unexported interface in the `main` package.